dogs.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>adopt-a-dog-spring2019-dogs</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+
+  <body>
+    <img class="headercolor" src="images/adoptadog-logo.svg" alt="adoptadog logo-header">
+<header>
+  <h1>Seattle's premiere dog adoption agency</h1>
+  <nav>
+    <a href="index.html">Home</a>
+    <a href="dogs.html">Dogs</a>
+    <a href="blog.html">Blog</a>
+    <a class="checkout" href="checkout.html">Checkout</a>
+  </nav>
+</header>
+
+    <section>
+      <h2>Here are all of our Dogs!</h2>
+    </section>
+
+    <article>
+      <h3>Murphy</h3>
```

Comment:
You're missing images for the dogs. You'll want to dump those in, even if they look terrible without stying